### PR TITLE
Bohdan Lysianskyi

### PR DESCRIPTION
**Pull request by HaroldFox17**
First task template

### CODE REVIEW

**Review by Dmitriy-NK — COMMENTED** [review]
File structure is OK. No additional change is required.

**Review by Dmitriy-NK — COMMENTED** [review]
По основным моментам все сделано хорошо. Из недочетов:
1. График 4 (в большом наборе графиков) построен с нормированием к полному числу пассажиров. Предполагается сначала разбиение на подмножества а потом для каждого подмножества построение графика.
2. Некорректные названия переменных, для гендерного распределения. Название не соответствует данным (для sman, swoman)
3. В том же разделе отсутствует график пропорционального распределения и просьба добавить несколько предложений по своей оценке после "Add your summary about recieved data for raw value counts and proportionally"
4. В следующем разделе (для Pclass) нет деления на выживших и не выживших, как вариант разделить на несколько панелей.
При построении ROC кривой предпочтительно делать график квадратным и содержащим прямую (0,0) (1,1)

**Review by Dmitriy-NK — COMMENTED** [review]
К первому заданию
По 3-у пункту просьба доработать график с долями единицы не от общего количества пассажиров а для каждого подмножества. 
Также есть описанная на лекции проблема с confusion matrix, по умолчанию sklearn.metrics.confusion_matrix считает метку 0 - положительной (это связано с возможностью классификации с многими метками) так собственный порядок можно задать в аргументе labels, в нашем случае [1,0]  [detailes](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)

**Review by meanrin — DISMISSED** [review]
Second task looks good
I liked that you implemented `LogisticRegression` using torch and not sklearn

**Review by Dmitriy-NK — APPROVED** [review]
First task LGTM. лучше было обновить вывод ячеек перед отправкой.
